File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    alpha = "abcdefghijklmnopqrstuvwxyz"
    
    for i in list(skip):
        alpha = alpha.replace(i, "")
        
    for a in s:
        answer += alpha[(alpha.find(a) + index) % len(alpha)]
    return answer

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    var answer = '';
    
    let alpha = "abcdefghijklmnopqrstuvwxyz"
    
    for (sk of skip) {
        alpha = alpha.replace(sk, "")
    }
    
    const n = alpha.length
    
    for (i of s) {
        answer = answer + alpha[(alpha.indexOf(i) + index) % n]
    }
    
    
    return answer;
}